flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/reconciler/deployment/ApplicationReconciler.java

```diff
@@ -302,6 +304,14 @@ public boolean reconcileOtherChanges(FlinkResourceContext<FlinkDeployment> ctx)
         return cleanupTerminalJmAfterTtl(ctx.getFlinkService(), deployment, observeConfig);
     }
 
+    private String getMsgRestartUnhealthy(ClusterHealthInfo clusterHealthInfo) {
+        if (clusterHealthInfo != null) {
```

Comment:
Why would the `clusterHealthInfo` be null here?

Reply:
It's because `getLastValidClusterHealthInfo` method in `ClusterHealthEvaluator` can return null